### DISCUSSION TITLE
I wonder  train data and validation data divide percentage

### DISCUSSION THREAD

**Discussion by cozy-hn** [autogluon/autogluon#3675] [Q&A]
As far as I know, if I don't set the tune data, the train data and validation data will be automatically divided. 
I wonder what percentage it is divided by.

**Comment by mglowacki100** [comment]
Hi @cozy-hn 
This value depends on number of rows, see description of paramereter: holdout_frac in https://auto.gluon.ai/stable/api/autogluon.tabular.TabularPredictor.fit.html
Here is function that is responsible for it:
https://github.com/autogluon/autogluon/blob/cb5098137b3bf604b4a800326533e597614a1507/core/src/autogluon/core/utils/utils.py#L185

**Reply by cozy-hn** [reply]
@mglowacki100 Thanks a lot!!
